core/src/main/java/org/jruby/RubyClass.java

```diff
@@ -1496,6 +1502,15 @@ public synchronized void reify(String classDumpDir, boolean useChildLoader) {
         }
     }
 
+    private String getReifiedJavaClassName() {
+        final String basePackagePrefix = "rubyobj.";
```

Comment:
This might be a good time to select a new package name that can fit in with JPMS, or perhaps to provide a way for users to configure the package name.